en/docs/reference/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-example.md

```diff
@@ -4,79 +4,92 @@ The AmazonSQS Inbound Endpoint allows you to connect to Amazon and consume messa
-           <parameter name="class">org.wso2.carbon.inbound.amazonsqs.AmazonSQSPollingConsumer</parameter>
-           <parameter name="inbound.behavior">polling</parameter>
-       </parameters>
-   </inboundEndpoint>   
-   ```   
-   **Sequence to process the message**
+2. Add a new **AmazonSQS inbound endpoint** by clicking `+` icon in the `Inbound Endpoints` and select `AmazonSQS WSO2 Listner Connector`.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/sqs-inbound.png" title="Creating inbound endpoint" width="800" alt="Creating inbound endpoint"/>
+
+3. Configure the AmazonSQS inbound endpoint as shown below and click **Create**.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/sqs-inbound-form.png" title="Creating AmazonSQS inbound endpoint form" width="800" alt="Creating AmazonSQS inbound endpoint form"/>
+
+    !!! Note
+        When creating the SMPP inbound endpoint you have two options in defining the injecting sequence and error sequence.
+        <br/>- **Automatic**: Click the checkbox *Automatically generate sequence*
+        <br/>- **Manual**: You can select already defined sequences as injecting and error sequences.
+
+    !!! Info
+        In this example, we are using `Automatic` option and that will create an injecting sequence named `AmazonSQSInboundEndpoint-inboundSequence` and an error sequence named `AmazonSQSInboundEndpoint-inboundErrorSequence`.<br/>
+
+    The source view of the created inbound endpoint is shown below.
+
+     ```xml
+     <?xml version="1.0" encoding="UTF-8"?>
+     <inboundEndpoint name="AmazonSQSInboundEndpoint" class="org.wso2.carbon.inbound.amazonsqs.AmazonSQSPollingConsumer" sequence="AmazonSQSInboundEndpoint-inboundSequence" onError="AmazonSQSInboundEndpoint-inboundErrorSequence" suspend="false">
+         <parameters xmlns="http://ws.apache.org/ns/synapse">
+             <parameter name="interval">2000</parameter>
+             <parameter name="sequential">true</parameter>
+             <parameter name="coordination">true</parameter>
+             <parameter name="destination">https://sqs.us-east-2.amazonaws.com/610968236798/eiconnectortestSQS</parameter>
+             <parameter name="contentType">text/plain</parameter>
+             <parameter name="accessKey">AKIAY4QELOL7GF35XBW5</parameter>
+             <parameter name="secretKey">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</parameter>
+             <parameter name="autoRemoveMessage">true</parameter>
+             <parameter name="attributeNames">All</parameter>
+         </parameters>
+     </inboundEndpoint> 
+     ```
+
+4. Implement the injecting sequence which process the message - `AmazonSQSInboundEndpoint-inboundSequence`. 
    
-   In this example, for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
+    In this example, for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
```

Comment:
```suggestion
    For simplicity, in this example you will just log the message. However, in a real-world use case, this could involve any type of message mediation.
```